src/crons/retry-error.rb

```diff
@@ -0,0 +1,89 @@
+#!/bin/ruby
+require 'yaml'
+require 'fileutils'
+require 'net/smtp'
```

Comment:
which part?  all of the requires appear to be used, I guess the email stuff could be chopped out since your doing it by hand on the command line anyway.